draft-ietf-quic-recovery.md

```diff
@@ -1022,13 +1021,31 @@ timers wake up late. Timers set in the past SHOULD fire immediately.
 Pseudocode for SetLossDetectionTimer follows:
 
 ~~~
+// Returns the earliest loss_time and the packet number
+// space it's from.  Returns 0 if all times are 0.
+GetEarliestLossTime():
+  time = loss_time[Initial]
+  space = Initial
+  for pn_space in [ Handshake, ApplicatonData ]:
+    if loss_time[pn_space] != 0 &&
+       (time == 0 || loss_time[pn_space] < time):
+      time = loss_time[pn_space];
+      space = Initial
+  return [time, space]
+
 SetLossDetectionTimer():
   // Don't arm timer if there are no ack-eliciting packets
   // in flight.
   if (no ack-eliciting packets in flight):
     loss_detection_timer.cancel()
     return
 
+  [loss_time, pn_space] = GetEarliestLossTime()
```

Comment:
In Python, you don't need square brackets to receive multiple return values.

And since we're not using the the `pn_space` here, we could write: `loss_time, _ = GetEarliestLossTime()`